target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-whw03.dts

```diff
@@ -0,0 +1,407 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Linksys WHW03 (Velop)";
+	compatible = "linksys,whw03", "qcom,ipq4019";
+
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_red;
+		led-running = &led_green;
+		led-upgrade = &led_red;
+	};
+
+	// The arguments rootfstype and ro are needed
+	// to override the default bootargs
+	chosen {
+		bootargs-append = " rootfstype=squashfs clk_ignore_unused";
```

Comment:
Please do not set `clk_ignore_unused`. You should configure the clocks correctly in the device tree and then this should not be needed.